scalameta/scalameta/shared/src/test/scala/scala/meta/tests/parsers/XmlSuite.scala

```diff
@@ -198,6 +198,12 @@ class XmlSuite extends ParseSuite {
     // format: on
   }
 
+  test("xml literals unit") {
+    val tree = term("<foo>{}</foo>")
+    assert(tree.show[Structure] == """Term.Xml(Seq(Lit.String("<foo>"), Lit.String("</foo>")), Seq(Term.Block(Nil)))""")
+    assert(tree.show[Syntax] == "<foo>{}</foo>")
```

Comment:
We need to remove the original tokens as is done here https://github.com/scalameta/scalameta/blob/2128ff7b233aafb797c8c1b9bc747a67e10efab4/scalameta/scalameta/shared/src/test/scala/scala/meta/tests/prettyprinters/SyntacticSuite.scala#L22 I propose we move this test case to SyntacticSuite right under the `xml literals` test.